Author: Henry-Nicolas Tourneur <debian@nilux.be>
Last-Update: 2020-09-25
Description: We are patching the Cargo.toml file to manually collapse features.
 The rationale is that this crate is aimed at supporting the squeekboard binary
 crate which only requires the wayland feature. 
 This is done to reduce empty binary packages generated by debcargo which are 
 considered problematic to this date.
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,13 +25,6 @@ version = "0.2"
 
 [dependencies.memmap]
 version = "0.7"
-optional = true
-
-[dependencies.xcb]
-version = "0.8"
-features = ["xkb"]
-optional = true
 
 [features]
-wayland = ["memmap"]
-x11 = ["xcb"]
+wayland = []
